airbyte_cdk/sources/streams/concurrent/cursor.py

```diff
@@ -442,3 +445,12 @@ def _evaluate_upper_safely(self, lower: CursorValueType, step: GapType) -> Curso
             return lower + step
         except OverflowError:
             return self._end_provider()
+
+    def should_be_synced(self, record: Record) -> bool:
+        record_cursor_value:  CursorValueType = self._extract_cursor_value(record)  # type: ignore  # cursor_field is converted to an InterpolatedString in __post_init__
```

Comment:
I think `self._extract_cursor_value` will raise a ValueError if the record does not have a cursor value. Can we add a test for that to validate this? If needed, we should update this code

Reply:
ok, I'll add one. Do we need to handle this error (and proceed record as `True`), or raising an Exception is ok ?

Reply:
Proceeding the record as true seems like the less disruptive thing to do